bin/steps/python

```diff
@@ -1,7 +1,11 @@
 #!/usr/bin/env bash
 
-set +e
-runtime-fixer runtime.txt
+# TODO: Replace runtime-fixer with a bash equivalent, since currently on rebuilds it
+# gets run with the cached Python install (rather than system Python), which means it
+# fails to run with glibc errors on stack downgrades (see spec/hatchet/stack_spec.rb).
+# To work around this for now, non-zero exit codes are ignored (most runtime.txt files
+# don't need fixing up, so the script not running won't normally matter).
+runtime-fixer runtime.txt || true
```

Comment:
Rather than use `set +e` around just this section (which is the last remaining misbehaving part of this file, now that the other error below was fixed in this PR), I've switched to the `|| true` approach for clarity.